### DISCUSSION TITLE
How can I make my searchable Select required?

### DISCUSSION THREAD

**Discussion by bsamworthdev** [filamentphp/filament#9643] [Help]
I've added a Select to my Form (see below) and I want to make it a required field. It works fine without  "->searchable()" but when I include it and try to save the form keeps saying the Title field hasn't been filled in even when it has.

Select::make('title')
    ->options(['Mr','Mrs','Ms'])
    ->searchable()
    ->label('Title')
    ->required(),

**Comment by sinnbeck** [comment]
Did you confirm the suggestion by hitting enter? If I do that it works just fine.

**Reply by bsamworthdev** [reply]
Yes it's the same whether I type the value and press enter or select it by clicking from the dropdown.

**Reply by sinnbeck** [reply]
I cannot recreate that. Make sure that your filament is up to date using `composer update` 
If that does not help, are you able to make a reproduction in a repo and share it ?

**Reply by bsamworthdev** [reply]
Ah I think it must be something else I've done in my setup. Thanks for checking- it was helpful to know it wasn't a global issue.

**Reply by sinnbeck** [reply]
You could try installing the official demo and add it to that to see if it works there. If it does you can compare the two code bases :)

https://github.com/filamentphp/demo
